SPECS/kata-containers-cc/kata-containers-cc.spec

```diff
@@ -204,14 +203,27 @@ install -D -m 0755 kata-monitor %{buildroot}%{coco_bin}/kata-monitor
 install -D -m 0755 kata-runtime %{buildroot}%{coco_bin}/kata-runtime
 install -D -m 0755 data/kata-collect-data.sh %{buildroot}%{coco_bin}/kata-collect-data.sh
 
-# Note: we deploy two configurations - the additional one is for policy/snapshotter testing w/o SEV SNP or IGVM
-install -D -m 0644 config/configuration-clh.toml %{buildroot}/%{defaults_kata}/configuration-clh.toml
+# We deploy 3 configurations:
+# configuration-clh-snp: production Kata-CC - IGVM & image, confidential_guest=true, sev_snp_guest=true
+# configuration-clh-snp-debug: debug Kata-CC - kernel & image, confidential_guest=true, sev_snp_guest=false
+# configuration-clh (symlinked to by configuration.toml): vanilla Kata - kernel & initrd, confidential_guest=false, sev_snp_guest=false
```

Comment:
The vanilla Kata and Kata-CC packages should imo be orthogonal to each other. We should not handle stuff that belongs to vanilla Kata in the kata-containers.spec. Am I missing something?

Reply:
One advantage of having `configuration-clh.toml` is that it makes it trivial to test vanilla code paths with the CC package, or to switch over both vanilla and CC runtimes to this package. WDYT?

Reply:
Question: With this, will both packages still be independent of each other? We had done prior work to detangle the kata-containers and kata-containers-cc SPECS and done other steps to ensure both packages can be installed without relying on the other package.

If `configuration-clh` as laid out here would introduce a dependency to anything that comes from the vanilla kata-containers package, I would abstain from doing this. I'm especially asking this because for Kata-CC we do _not_ build an initrd to my knowledge. So, where would the initrd come from?

Reply:
Good point about the initrd, we'll be making the switch to an image soon.

Re: interdependence, both packages are still independent with this since vanilla gets installed in /usr and CC in /opt.

Reply:
@ms-mahuber, while you were out, we discussed adopting the Kata main branch approach of setting up for Vanilla and SNP in a single shot. @sprt, let's review our ideas with Manuel too.